UploadFile/Table/Tab50410.BCPUpload.al

```diff
@@ -0,0 +1,56 @@
+table 50410 "BCP Upload"
+{
+    DataClassification = CustomerContent;
+    InherentPermissions = RIMDX;
+
+    fields
+    {
+        field(1; "BCP ID"; Integer)
```

Comment:
You don't need to use affixes in your object.

Reply:
You mean objects my custom objects? both the object names, fields etc? i don't need to use affixes?
does that mean i should only use it when extending base objects?

Reply:
Ask yourself why affixes are necessary in principle? 

To avoid conflicts with standard objects.

The name of your table can, in theory, conflict with the name of another standard table? The answer is yes.

The name of your fields in your table can conflict with standard fields? The answer is no.

 Follow this rule.

Reply:
alright i'll keep this in mind